xml/run-config-processor.xml

```diff
@@ -12,26 +12,32 @@
   run with the changes that were applied.
  </para>
  <procedure>
+  <step>
+   <para>
+    If nodes have been added after deployment, generate a hosts file.
```

Comment:
If the nodes have been added after a deployment, a hosts file is generated:* (or something to that effect?)